packages/mui-system/src/cssContainerQueries/cssContainerQueries.ts

```diff
@@ -98,7 +99,8 @@ export default function cssContainerQueries<T extends { breakpoints: Breakpoints
         return result
           .replace('not all and ', '')
           .replace('min-width:', 'width<')
-          .replace('max-width:', 'width>');
+          .replace('max-width:', 'width>')
+          .replace('and', 'or');
```

Comment:
What case does this cover?

Reply:
Please see the test case https://github.com/mui/material-ui/pull/42434/files#r1623064430